ledger/eval_simple_test.go

```diff
@@ -224,92 +224,84 @@ func TestMiningFees(t *testing.T) {
 		dl := NewDoubleLedger(t, genBalances, cv, cfg)
 		defer dl.Close()
 
-		tooBig := basics.Address{0x01, 0x011}
-		tooSmall := basics.Address{0x01, 0x022}
-		smallest := basics.Address{0x01, 0x033}
-		biggest := basics.Address{0x01, 0x044}
-
+		proposer := basics.Address{0x01, 0x011}
 		const eFee = 3_000_000
-		dl.txns(
-			&txntest.Txn{Type: "pay", Sender: addrs[1],
-				Receiver: tooBig, Amount: eFee + 100_000_000*1_000_000 + 1},
-			&txntest.Txn{Type: "pay", Sender: addrs[1],
-				Receiver: tooSmall, Amount: eFee + 100_000*1_000_000 - 1},
-			&txntest.Txn{Type: "pay", Sender: addrs[1],
-				Receiver: smallest, Amount: eFee + 100_000*1_000_000},
+		dl.txn(
 			&txntest.Txn{Type: "pay", Sender: addrs[1],
-				Receiver: biggest, Amount: eFee + 100_000_000*1_000_000},
+				Receiver: proposer, Amount: eFee + 50_000_000*1_000_000 + 1},
```

Comment:
Let me get this straight: in real-life evaluation, this account would not receive incentives, since after the keyreg, its balance is 1 microAlgo over the 50M limit.

But this test doesn't use the real agreement code that would put that limit in place. Instead, in `ledger/simple_test.go`, it always calls `vb.WithProposal` with eligible=true if mining is enabled.

So the actual balance isn't a factor here

Reply:
Correct. We can't test `agreement` code here, so we can't test balance eligibility.  It'll have to move into e2e tests.